tests/test_buffer.py

```diff
@@ -436,6 +437,11 @@ def test_SimpleArray_from_ndarray(self):
         sarr_from_cpp = modmesh.SimpleArrayFloat64(shape=(2, 3, 4))
         self.assertFalse(sarr_from_cpp.is_from_python)
 
+        shape = (2, 2, 2, 2)
+        np_sarr = np.ndarray(shape=shape, dtype=np.float64)
```

Comment:
Use string for the argument `dtype`, i.e., write `dtype='float64'`.  modmesh uses a convention to use string for numpy `dtype` argument.

Reply:
OK I'll fix it